Sasquatch/Sasquatch/Info.plist

```diff
@@ -16,14 +16,12 @@
 	<string>APPL</string>
```

Comment:
We should maintain two different info.plist for URL scheme. Probably it is good to remove the one in Sasquatch/